exercises/practice/rational-numbers/src/RationalNumbers.hs

```diff
@@ -0,0 +1,46 @@
+module RationalNumbers
+(Rational,
```

Comment:
As there is a smart constructor `rational`, perhaps the 'dumb' `Rational` should not be exported.

Reply:
Having Rational exported allows pattern matching which is beneficial for simple data with defined internal structure (cf. Complex).

Not exporting data constructor directly it is beneficial if a smart constructor shall assure some kind of internal consistency. This might be a case also here if we wanted to assure correct rationals - see also [this comment](https://github.com/exercism/haskell/pull/1193#issuecomment-1898003150), but this goes beyond the scope of this exercise.